frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignDrawer/EditFieldDrawer/edit-fieldtype/common/useEditFieldForm.ts

```diff
@@ -168,10 +167,9 @@ export const useEditFieldForm = <FormShape, FieldShape extends FormField>({
     setToInactive()
   }, [setToInactive])
 
-  useDebounce(
+  useEffect(
     () => handleChange(transform.output(clonedWatchedInputs, field)),
-    300,
-    Object.values(clonedWatchedInputs),
+    [clonedWatchedInputs, field, handleChange, transform],
```

Comment:
does this work properly? was put in as it seemed to not update the preview without destructuring the inputs (since they looked like the same reference to react)

Reply:
i think youre right, saving fields seem to break. updated to simply use 0 delay